lib/diagnostic.js

```diff
@@ -38,19 +46,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: "Mr.Paws",
+  species: "dog",
+  breed: "Siberian Husky",
+  vocalize: function(vocalization){
+    let vocalization = "bark bark"
```

Comment:
you are declaring vocalization as variable here AND passing it in as an argument which will break the code